stock_picking_procure_method/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Procure Method",
     "summary": "Allows to force the procurement method from the picking",
-    "version": "14.0.1.0.0",
+    "version": "16.0.0.0.0",
```

Comment:
@pedrobaeza This one is not correct

Reply:
Ouch, I didn't notice. I will do direct commit, as it will trigger the same the wheel creation.